client/src/components/shared/StarRating.jsx

```diff
@@ -27,15 +22,7 @@ class StarRating extends React.Component {
   }
 
   getImage(rating) {
-    if (rating === 25) {
```

Comment:
Why so many if statements? It seems that these could be stored in an array/object and referenced dynamically instead of each being an isolated variable name, but maybe `import` has restrictions I don't know about.

Reply:
With this few of items, it was ambiguous whether it was better to do a more switch-statement-like solution or have things less hard-coded. Seeing it pointed out, I guess this can be stored as a hash table that maintains clarity. I'll change that!

Reply:
Wait, actually that change isn't needed unless we go back to that state of webpack bundling images. Since we used Express to serve the images, I went back to the even simpler operation of inserting the rating # into a URL string that corresponds with the appropriate image.